packages/ndla-image-search/src/ImageSearch.js

```diff
@@ -352,7 +356,8 @@ class ImageSearch extends React.Component {
   }
 
   render() {
-    const { searchPlaceholder, searchButtonTitle, useImageTitle } = this.props;
+    const { searchPlaceholder, searchButtonTitle, useImageTitle, showMetaImageCheckbox, useAsMetaImageLabel } =
```

Comment:
Burde nye felt ha default-verdier for bakoverkompabilitet? Bildesøket brukes også i h5p-editoren så vi må passe på at ikkje deira bruk av dette knekkes med ny funksjonalitet.

Reply:
La til default value `false` på visning av checkbox. Det skal være bakoverkompatibelt nå. Valg av metabilde i audio fungerer feks som normalt.